### PR TITLE
Moved custom Institutional configs to nf-core configs on github

### PR DESCRIPTION
**Pull request by drpatelh**
Many thanks to contributing to nf-core/tools!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [x ] This comment contains a description of changes (with reason)
 - [x] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] Documentation in `docs` is updated
 - [x] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/tools/tree/master/.github/CONTRIBUTING.md

We can now centrally store and maintain custom configs that are specific to individual Institutions. Config files have been moved to `nf-core/configs` and will be pulled by nextflow before pipeline execution.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # nf-core/tools: Changelog
 
+## v1.3dev
```
Comment (apeltzer) [inline CHANGELOG.md]:
```suggestion
## v1.4dev
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+* Institutional custom config profiles moved to github `nf-core/configs`
+    * These will now be maintained centrally as opposed to being shipped with the pipelines in `conf/`
+    * Load `base.config` by default for all profiles
+    * Removed profiles named `standard` and `none`
+    * Added parameter `--igenomesIgnore` so `igenomes.config` isnt loaded if parameter clashes are observed
```
Comment (apeltzer) [inline CHANGELOG.md]:
```suggestion
    * Added parameter `--igenomesIgnore` so `igenomes.config` isn't loaded if parameter clashes are observed
```

File: CHANGELOG.md (hunk 1 above)
Comment (drpatelh) [reply CHANGELOG.md]: Sorry. Thats exactly what I meant!

Comment (drpatelh) [reply CHANGELOG.md]: How do you make these changes after merging @apeltzer? My naive brain is trying to work it out!

File: CHANGELOG.md (hunk 2 above)
Comment (drpatelh) [reply CHANGELOG.md]: Agreed!

File: CHANGELOG.md (hunk 1 above)
Comment (apeltzer) [reply CHANGELOG.md]: My bad - I've merged before really thinking as I thought the commit was fine and good (expect for these smallish typos). I guess just make a new PR with just these two then and I'll merge it in... sorry about that. 

Normally you can simply click "accept" and thats it... :-)

Comment (drpatelh) [reply CHANGELOG.md]: No worries. I have fixed this in https://github.com/nf-core/tools/pull/214 whilst I was dealing with another issue.

**Comment by ewels** [comment]
Super nice work @drpatelh!

Please can you describe this change in `CHANGELOG.md` and also add some documentation about how it works in the template pipeline docs?

**Comment by drpatelh** [comment]
Sorry got started with some of this and had to prep for presenting the pipeline to our User Committee. Back to reality!

I did put some description in the CHANGELOG.md. Did I miss something?
https://github.com/nf-core/tools/blob/dev/CHANGELOG.md

Assuming you mean adding the parameters in here:
https://github.com/nf-core/tools/blob/dev/nf_core/pipeline-template/%7B%7Bcookiecutter.name_noslash%7D%7D/docs/usage.md

And a description here:
https://github.com/nf-core/tools/blob/dev/nf_core/pipeline-template/%7B%7Bcookiecutter.name_noslash%7D%7D/docs/configuration/adding_your_own.md

**Comment by drpatelh** [comment]
https://github.com/nf-core/tools/pull/219

Added some description as requested and a bunch of other stuff I had noticed whilst writing `nf-core/atacseq`

**Comment by ewels** [comment]
You did indeed add something to the changelog, correct. Not sure how I missed that, sorry!

For the docs I more meant something specifically describing that the configs are fetched from a separate repository and people can add their clusters their so that they're available for all pipelines.

**Comment by ewels** [comment]
Note to self: refresh half-written replies on GitHub before posting.

This is great! I will stop talking now.

**Pull request merged**